public/javascripts/overlays/admin.js

```diff
@@ -318,7 +318,27 @@ function sendUpdate() {
         'stateRight': window.stateRight
     };
     socket.emit('update overlay', data);
-    toastNotify();
+    toastNotify('Overlay data updated.');
+}
+
+function changeLayout() {
+    var layout = document.getElementsByClassName('selectedLayout')[0];
+    var data = {
+        'layout': layout ? layout.value : 'empty-layout',
+        'background': document.getElementById('layout-background').value
```

Comment:
I see you're doing a null check above but not here. Is it okay if this `background` field is `null`?